params/chromium-params.md

```diff
@@ -28,6 +28,7 @@ Chromium's implementation assigns the following values:
 | [Max event-level channel capacity for event sources][] | [6.5 bits][max event-level channel capacity for events value] |
 | [Max unique attribution scope sets per navigation][] | [1][max unique attribution scope sets per navigation value] |
 | [Max trigger-state cardinality][] | [4,294,967,295 (max unsigned 32-bit integer)][max trigger-state cardinality value] |
+| [Max event-level channel capacity for navigation sources with attribution scopes][] | [11.55 bits][max event-level channel capacity for navigations with attribution scopes value] |
```

Comment:
We also need one for event sources with attribution scopes and the value is 6.5.

Reply:
@linnan-github  Just updated to include event source check